src/odemis/driver/test/lakeshore_test.py

```diff
@@ -111,12 +111,15 @@ def test_heating_test(self):
         Test heating
         """
         old = self.dev.targetTemperature.value
+        # set a target temperature 1 degree higher and start heating
         self.dev.targetTemperature.value = self.dev.temperature.value + 1
         self.dev.heating.value = 3  # fast heating
+        # wait to heat up and then check if the temperature rose by a degree
         time.sleep(30)
-        self.assertAlmostEqual(self.dev.temperature.value, self.dev.targetTemperature.value)
+        self.assertAlmostEqual(self.dev.temperature.value, self.dev.targetTemperature.value, places=0)
```

Comment:
"places=0" is not very readable/clear. I'd suggest to replace with `delta=0.5` .